docker-pushmi-pullyu

```diff
@@ -12,7 +12,8 @@ Usage:	$0 [OPTIONS] [USER@]HOST NAME[:TAG] [NAME[:TAG]...]
 Push images directly to a remote host (without a separate registry)
 
 Options:
-  -s, --ssh_opts string   Specify additional ssh arguments (e.g. --ssh_opts \"-i private.pem -C\")"
+  -s, --ssh_opts string   Specify additional ssh arguments (e.g. --ssh_opts \"-i private.pem -C\")
```

Comment:
I'm planning to change the long form of this option from `--ssh_opts` to `--ssh-opts` for consistency's sake, but will do so in a separate pull request.

Reply:
See #27.